add new event and speakers for Vue.js Israel meetup on 21 May 2025

AlexNechayev opened the pull request:
- Updated event details for "Vue with Imagen" including metadata, agenda, and sponsors.
- Added new speakers: Narkis Feldman and Alex Nechayev with their respective details.
- Introduced new talks related to the event, enhancing the content for attendees.

dimshik100 commented on content/talks.json:
```diff
@@ -493,5 +493,50 @@
+    "talkPoster": {
+      "src": "/speaker-posters/Vue.js-Israel-speakers-narkis-feldman-21.05.png"
+    },
+    "videos": [""],
```
Missing video. please add

dimshik100 commented on content/speakers.json:
```diff
@@ -181,5 +181,19 @@
+    "name": "Narkis Feldman",
+    "title": "Front-End Team Lead",
+    "image": "/speaker-pictures/Narkis Feldman.jpg",
+    "socialLinks": [],
```
Please add linkedin

AlexNechayev committed:
add new event and speakers for Vue.js Israel meetup on 21 May 2025

- Updated event details for "Vue with Imagen" including metadata, agenda, and sponsors.
- Added new speakers: Narkis Feldman and Alex Nechayev with their respective details.
- Introduced new talks related to the event, enhancing the content for attendees.
